tests/TestRunner/app/Marshalling/ObjCTypesTests.js

```diff
@@ -204,6 +204,14 @@ describe(module.id, function () {
         expect(interop.handleof(buffer)).toBe(data.bytes);
     })
 
+    it("MethodWithNSDecimalNumber", function () {
+        expect(NSDecimalNumber.maximumDecimalNumber() instanceof NSDecimalNumber).toBe(true);
```

Comment:
Technically it'd be the same, but why'd you choose this over `expect(NSDecimalNumber.maximumDecimalNumber().toEqual(jasmine.any(NSDecimalNumber))`?